frontend/src/app/app-routing.module.ts

```diff
@@ -46,7 +46,8 @@ const appRoutes: Routes = [
       title: 'US Forest Service Open Forest',
```

Comment:
Food for thought (and future work), it might be easier to grok the routes and authentication if we can break them up into groups depending on the required authentication AND have specific `CanActivate` implementations for each group...

Reply:
Was replying to above before I saw this. Yes and Yes, wholeheartedly agree and with @aaronburk blessing, will begin investigating creating separate admins based off of requested routes.